py-polars/tests/unit/test_config.py

```diff
@@ -196,7 +196,7 @@ def test_set_tbl_rows() -> None:
         "╞═════╪═════╪═════╡\n"
         "│ 1   ┆ 5   ┆ 9   │\n"
         "│ 2   ┆ 6   ┆ 10  │\n"
-        "│ 3   ┆ 7   ┆ 11  │\n"
+        "│ …   ┆ …   ┆ …   │\n"
```

Comment:
Technically we can print this row instead of printing dots and keep the same number of lines printed, but it feels off to print 4 rows of data when the limit is set to 3.